vqe.py

```diff
@@ -0,0 +1,84 @@
+import networkx as nx
+import numpy as np
+
+from graphix import Circuit
+from graphix.simulator import PatternSimulator
+
+# Define the Hamiltonian for the VQE problem (Example: H = Z0Z1 + X0 + X1)
```

Comment:
could you change comments to format below, so they render well on sphinx-gallery?
```python
# %%
# Define the .. 
```